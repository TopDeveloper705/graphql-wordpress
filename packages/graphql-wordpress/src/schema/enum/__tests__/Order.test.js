import ORDER from 'enum/Order';

describe('Test enum values', () => {
  test('ORDER values', () => {
    expect(ORDER.getValues()).toMatchSnapshot();
  });
});
