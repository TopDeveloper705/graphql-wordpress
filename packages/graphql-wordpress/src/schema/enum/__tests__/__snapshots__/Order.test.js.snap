// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test enum values ORDER values 1`] = `
Array [
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "ASC",
    "value": "asc",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "DESC",
    "value": "desc",
  },
]
`;
